src/Home.tsx

```diff
@@ -166,9 +166,19 @@ const Home = function Home() {
       setPendingTransactions(response.pendingTransactions);
     });
 
-    UserState.getPendingTxStore().then((response : PendingTransactionStore) => {
-      setPendingTransactions(response.pendingTransactions);
-    });
+    // window.addEventListener('load', () => {
```

Comment:
Remove this commented code

Reply:
I will eventually remove, but that's my starting point for trying to figure out how to do the realtime updates when a transaction lands or reverts.